### ISSUE TITLE
Contact Center Review for [Authenticated Experience My VA - Onsite Notifications & My VA Redirect]

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#45091]
# Request the Contact Center Review 

## What this form is for
Use this template to request a Contact Center review of your product.

Please review the Self-service Product Guide Template [link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/self-service-product-guide-template.md) and the Contact Center Review How to [link](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/contact-center/request-contact-center-review.md)
 

## Description
We are updating the My VA product guide to include two new features that are launching within the same timeframe. We've updated the main "Navigation" section of the product guide to explain how users will soon be immediately redirected to My VA post login when they do so from the homepage on VA.gov. We've also added a section to the product guide for our new Notifications on My VA. This is the MVP for notifications and only includes functionality for debt notifications. 

[My VA Redirect](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/my-va/my-va-redirect#my-va-redirect-initiative-brief)
[Onsite Notifications (MVP)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/notifications/vagov-notifications/mvp/README.md#project-outline-on-site-notifications-mvp)

Finally, we made a minor update to the Benefit Payments and Debts section of the product guide to cover some changes to conditional behavior we corrected recently.

### Launch Date
My VA redirect: ~August 31 (begin phased launch)
Onsite Notifications: ~August 31 (begin phased launch)

### Version Number
version 4

### Product Guide
[MyVA_ProductGuide_V4_071822.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/9222882/MyVA_ProductGuide_V4_071822.docx)

## Acceptance Criteria
- [ ] Platform: Contact Center QA has been completed 
- [ ] Platform: Contact Center sends artifacts to appropriate contact centers
- [ ] Platform: [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886) is updated

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Contact-center-review.1782317061.html) or the [Contact Center Review Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Contact-center-review.1782317061.html) on Platform website for more information about the Collaboration Cycle.

**Comment by ajakabcin** [comment]
hi @Kimberley2019 I was hoping to check in on our submission - has there been any progress on the review? Thank you!

**Comment by Kimberley2019** [comment]
Hi @ajakabcin my sincerest apologies!  I have included Steven Tarlow, our technical writer, to review this guide.

@starlow00  can we move this up the priority list? It's my fault that it hasn't been reviewed as of yet.

**Comment by starlow00** [comment]
Will do, @Kimberley2019

**Comment by ajakabcin** [comment]
No problem @Kimberley2019! Thank you for expediting : )

**Comment by starlow00** [comment]
Hi @ajakabcin. My review draft is attached. Once you send me back a copy for VEO, I'll make sure the page numbering in the TOC and Table of Figures match up before sending on.
[MyVA_ProductGuide_V4_071822_srt.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/9329962/MyVA_ProductGuide_V4_071822_srt.docx)

**Comment by ajakabcin** [comment]
Hi @starlow00! Thank you for the review on this! Some notes below:

1. I corrected the links you found that were going to staging - great catch!
2. The other links that weren't working require the user to be authenticated so the url is more there for reference as the call center won't be able to see that anyway.
3. I noticed that a lot of places had "MyVA" with no spaces. The product is actually "My VA" with a space, so I did a find and replace for all instances of those.

This is all set from our perspective. Thank you again! (attached with my revisions saved)
[MyVA_ProductGuide_V4_071822_srt_agj.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/9365840/MyVA_ProductGuide_V4_071822_srt_agj.docx)
